frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.console.rdp.scenario.ts

```diff
@@ -138,17 +137,13 @@ describe('KubeVirt VM console - RDP', () => {
        */
 
       await vm.navigateToDetail();
-      // eslint-disable-next-line no-console
-      console.log(
-        'Waiting for static IP to be reported by the guest-agent (can take up to several minutes ...)',
-      );
       // Waiting for instllation & start of the guest-agent and reporting the static IP back
```

Comment:
can we add this statement `(can take up to several minutes ...)` as a comment here?

Reply:
NP